Makefile

```diff
@@ -0,0 +1,17 @@
+PHONY: infection phpstan
+./infection.phar:
+	wget https://github.com/infection/infection/releases/download/0.8.1/infection.phar
+	chmod a+x ./infection.phar
+
+./infection.phar.pubkey:
```

Comment:
Until all phars are completely scoped, using phar distributions is out of question due to autoloaded dependency conflicts

Reply:
Infection, in the versions below `0.9.0` does not autoload any code.